src/math/fields/fields.zig

```diff
@@ -601,6 +604,28 @@ pub fn Field(comptime F: type, comptime modulo: u256) type {
             );
         }
 
+        pub fn fromSignedInt(comptime T: type, signed: T) Self {
```

Comment:
Can we add unit tests in the dedicated file in order to confirm the validity of the method?